`pause!` macro requires dependent crates to explicitly depend on tokio

**Description**
The macro `pause!` uses `tokio::time::sleep().await` internally but does not re-export the `tokio` crate from `hermes_five`.
As a result, any crate that depends on hermes_five and uses `pause!` must also declare tokio in its own dependencies, causing unnecessary dependency pollution and compilation errors if omitted.

**Steps to Reproduce**

1. Create a minimal crate depending on hermes_five without declaring tokio:
2. Add this code in main.rs

``` rust
use hermes_five::hardware::{Board, BoardEvent};
use hermes_five::pause;

#[hermes_five::runtime]
async fn main() {
    let board = Board::run();

    board.on(BoardEvent::OnReady, |board: Board| async move {
        pause!(5000); // <-- Compile error here if tokio is missing
        Ok(())
    });
}
```
Compile the project, observe the error:
``` bash
error[E0433]: failed to resolve: use of undeclared crate or module `tokio`
```

**Expected Behavior**
The `pause!` macro should work without requiring dependent crates to declare tokio explicitly.

**Additional Notes**
- This can be fixed by re-exporting tokio from hermes_five and updating the macro to use `$crate::tokio`.
- This is a backward-compatible change.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -40,3 +40,8 @@ jobs:
         run: cargo test --workspace --verbose --color=always
         env:
           RUST_BACKTRACE: 1
+
+      - name: Test compiling external project using Hermes-Five library
+        run: |
+          cd hermes-five/tests/compile_tests
+          cargo check

--- a/hermes-five/src/lib.rs
+++ b/hermes-five/src/lib.rs
@@ -80,6 +80,8 @@
 #[cfg(test)]
 extern crate self as hermes_five;
 
+pub extern crate tokio;
+
 pub mod animations;
 pub mod devices;
 pub mod errors;

--- a/hermes-five/src/utils/task.rs
+++ b/hermes-five/src/utils/task.rs
@@ -175,7 +175,7 @@ where
 #[macro_export]
 macro_rules! pause {
     ($ms:expr) => {
-        tokio::time::sleep(tokio::time::Duration::from_millis($ms as u64)).await
+        $crate::tokio::time::sleep($crate::tokio::time::Duration::from_millis($ms as u64)).await
     };
 }
 

--- a/hermes-five/tests/compile_tests/Cargo.toml
+++ b/hermes-five/tests/compile_tests/Cargo.toml
@@ -1,0 +1,10 @@
+[package]
+name = "compile_tests"
+description = "Tests use of Hermes-Five as a library"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+hermes-five = { path = "../../" }

--- a/hermes-five/tests/compile_tests/src/main.rs
+++ b/hermes-five/tests/compile_tests/src/main.rs
@@ -1,0 +1,6 @@
+use hermes_five::pause;
+
+#[hermes_five::runtime]
+async fn main() {
+    pause!(500);
+}

